### PR TITLE
Use global libraries instead of hidden code

### PR DESCRIPTION
**Pull request by daviroo**
Paired with https://github.com/saving-satoshi/saving-satoshi-backend/pull/33.  They need to be reviewed together.

### CODE REVIEW

**Comment by 0tuedon** [comment]
Hey @daviroo do you have permission to add me to the backend repo not sure i have access

**Comment by 0tuedon** [comment]
Hey @daviroo do you have permission to add me to the backend repo not sure i have access

File: content/lessons/chapter-4/address-3.tsx [hunk 1]
```diff
@@ -74,10 +73,10 @@ function hashToAddress(hash) {
 
       return [true, undefined]
     },
-    hiddenRange: [1, 0, 160, 0],
+    hiddenRange: [1, 0, 1, 0],
```
Comment (benalleng) [inline content/lessons/chapter-4/address-3.tsx:76]: I think you can simply remove the hidden range if you don't want it anymore

**Review by benalleng — CHANGES_REQUESTED** [review]
Other than removing the uneccessary hidden range it's looking good!

**Comment by daviroo** [comment]
> Hey @daviroo do you have permission to add me to the backend repo not sure i have access

You should be good now

**Review by benalleng — APPROVED** [review]

**Pull request merged**